src/boxed.rs

```diff
@@ -205,6 +205,14 @@ macro_rules! glib_boxed_wrapper {
             }
         }
 
+        #[doc(hidden)]
+        impl $crate::translate::FromGlibPtrBorrow<*const $ffi_name> for $name {
+            #[inline]
+            unsafe fn from_glib_borrow(ptr: *const $ffi_name) -> Self {
+                $name($crate::translate::from_glib_borrow(ptr as *mut $ffi_name))
```

Comment:
You can also remove the `$name(...)` part here :)

Reply:
It's everywhere! :o